roles/container-engine/containerd/templates/config-v2.toml.j2

```diff
@@ -0,0 +1,101 @@
+version = 2
```

Comment:
Hi @0ekk,

Would naming the file "config-v1.toml" make more sense since it's for containerd v1? Also, we could add some comments like "This is for containerd v1" to make it more developer-friendly.

Reply:
/full-test